reciprocalspaceship/utils/rfree.py

```diff
@@ -66,8 +76,22 @@ def copy_rfree(dataset, dataset_with_rfree, inplace=False):
     if not inplace:
         dataset = dataset.copy()
 
-    dataset["R-free-flags"] = 0
-    dataset["R-free-flags"] = dataset["R-free-flags"].astype(MTZIntDtype())
+    if rfree_key is not None:
+        if rfree_key not in dataset_with_rfree.columns:
+            raise ValueError(
+                f"""Supplied dataset_with_rfree contains no column {rfree_key}"""
+            )
+    elif "R-free-flags" in dataset_with_rfree.columns:
+        rfree_key = "R-free-flags"
+    elif "FreeR_flag" in dataset_with_rfree.columns:
+        rfree_key = "FreeR_flag"
+    else:
+        raise ValueError(
+            """Failed to automatically find r-free flags in dataset_with_rfree. Please supply a custom_rfree_key"""
```

Comment:
sorry small thing -- do you mind changing `custom_rfree_key` here too in the ValueError?

Reply:
Ah missed that, my bad